packages/compass-components/package.json

```diff
@@ -43,13 +43,15 @@
     "@leafygreen-ui/code": "^16.0.2",
     "@leafygreen-ui/combobox": "^11.0.2",
     "@leafygreen-ui/confirmation-modal": "^6.0.2",
+    "@leafygreen-ui/descendants": "^2.1.0",
     "@leafygreen-ui/emotion": "^4.0.9",
     "@leafygreen-ui/guide-cue": "^7.0.2",
     "@leafygreen-ui/hooks": "^8.3.4",
     "@leafygreen-ui/icon": "^13.1.2",
     "@leafygreen-ui/icon-button": "16.0.2",
     "@leafygreen-ui/info-sprinkle": "^4.0.2",
     "@leafygreen-ui/leafygreen-provider": "^4.0.2",
+    "@leafygreen-ui/lib": "^15.2.0",
```

Comment:
We now have two versions of lib package in compass-components, 14 and 15. Is it required to pull this dependency here?

Reply:
Got it, that's fair, yeah